### ISSUE TITLE
Put a copyright and license notice in each file ?

### ISSUE THREAD

**Issue by agilare** [agilare/ladecadanse#70]
After #60 done, with a AGPL license defined, decide if a corresponding license advice should inserted in each source file, and in which format (complete or short), like described in https://github.com/agilare/ladecadanse/issues/60#issuecomment-1816679873
First, just the LICENSE file and "Licence" part in README should be enough

**Comment by VonUniGE** [comment]
According to [How to Use GNU Licenses for Your Own Software
](https://www.gnu.org/licenses/gpl-howto.en.html)
> This involves adding two elements to each source file of your program: a copyright notice (such as “Copyright 1999 Terry Jones”), and a statement of copying permission, saying that the program is distributed under the terms of the GNU General Public License (or the Lesser GPL, or the Affero GPL).

**Comment by agilare** [comment]
Yes it seems better but according to an IA it isn't necessary and can be lightened : https://chat.openai.com/share/825be2ed-0b17-46ad-bcde-b96d63076e9b
